app/components/Autopay/Autopay.js

```diff
@@ -23,11 +27,16 @@ const Autopay = props => (
         </Panel.Header>
         <Panel.Body css={{ 'overflow-y': 'overlay', 'overflow-x': 'hidden' }}>
           <AutopayList mx={4} />
+          <Heading.h1 mt={4} mx={4}>
+            <FormattedMessage {...messages.merchant_list_title} />
+          </Heading.h1>
+          <AutopayActions mx={4} my={3} />
+          <AutopayMerchantList mx={4} />
         </Panel.Body>
       </Panel>
       <AutopayCreateModal />
     </>
   </ThemeProvider>
 )
 
-export default Autopay
+export default injectIntl(Autopay)
```

Comment:
Looks like `injectIntl` isn't needed since the `intl` prop isn't being used anywhere.